modules/gitops-release-notes-1-9-1.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-9-1_{context}"]
+= Release notes for {gitops-title} 1.9.1
+
+{gitops-title} 1.9.1 is now available on {product-title} 4.12 and 4.13.
+
+[id="errata-updates-1-9-1_{context}"]
+== Errata updates
+
+=== RHSA-2023:3591- {gitops-title} 1.9.1 security update advisory 
+
+Issued: 2023-07-10
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:3591[RHSA-2023:3591]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="new-features-1-9-1_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the bundled Argo CD has been updated to version 2.7.6.
+
+[id="fixed-issues-1-9-1_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, Argo CD was becoming unresponsive when there was an increase in namespaces and applications. This update fixes the issue by removing a deadlock. The deadlock occurs when two functions are competing for resources. Now, users should not experience crashes or unresponsiveness when there is an increase in the number of namespaces or applications. link:https://issues.redhat.com/browse/GITOPS-2782[GITOPS-2782]
+
+* Before this update, `argocd-application-controller` would suddenly stop working when resychronizing. This update fixes the issue by adding logic to prevent a cluster cache deadlock. Now, users should not experience the deadlock situation and applications should resynchronize successfully. link:https://issues.redhat.com/browse/GITOPS-2880[GITOPS-2880]
+
+* Before this update, There was an RSA key mismatch for `known_host` in `argocd-ssh-knownhosts-cm`. This update fixes the issue by matching the RSA key with the upstream project. Now, users should be able to use the default RSA keys on default deployments. link:https://issues.redhat.com/browse/GITOPS-3042[GITOPS-3042]
+
+* Before this update, reconciliation timeout setting in the `argocd-cm` configmap was not being correctly applied to the controller. This update fixes the issue by correctly reading and applying the reconciliation timeout setting from the `argocd-cm` configmap. Now, users can modify the reconciliation timeout from the `appSync` setting without a problem. link:https://issues.redhat.com/browse/GITOPS-2810[GITOPS-2810]
```

Comment:
```suggestion
* Before this update, reconciliation timeout setting in the `argocd-cm` config map was not being correctly applied to the Argo CD application controller. This update fixes the issue by correctly reading and applying the reconciliation timeout setting from the `argocd-cm` config map. Now, users can modify the reconciliation timeout from the `AppSync` setting without a problem. link:https://issues.redhat.com/browse/GITOPS-2810[GITOPS-2810]
```

Reply:
The setting in the CR is shown as `AppSync`.

Reply:
The names of the instances for all the above RNs were suggested based on the guideline:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting
which was also pointed out by @Srivaralakshmi here:
https://github.com/openshift/openshift-docs/pull/62126#discussion_r1255517596

Reply:
Yes, I Missed that in my last update, and i was going to address in my next commit.